### PR TITLE
adding 'arson.dino.icu'

### PR DESCRIPTION
**Pull request by arsoninstigator**
# [Adding] `arson.dino.icu`

## Description

This will be used as my personal website. I intended for dino.icu to be the domain but accidentally added my request to hackclub.com.yaml first :/ (oops)

### CODE REVIEW

**Comment by ajhalili2006** [comment]
As someone who maintains a package in Alpine Linux, the aports maintainers told me to do edits and amend the commit (`git commit --amend`) when situtations like this arises. If you published your personal branch/fork, you need to do `git push --force-with-lease`.

File: hackclub.com.yaml [hunk 1]
```diff
@@ -426,6 +426,10 @@ archived.guide:
   ttl: 600
   type: CNAME
   value: hackclub-guide.netlify.com.
+arson:
```
Comment (Muirrum) [inline hackclub.com.yaml:429]: Please remove from hackclub.com

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (arsoninstigator) [reply hackclub.com.yaml:429]: > Please remove from hackclub.com

i have removed it from hackclub.com.yaml and added the value it to dino.icu.yaml in it's correct alphabetical spot

**Comment by jaspermayone** [comment]
@arsoninstigator you should be all good now!

**Review by jaspermayone — APPROVED** [review]

**Pull request merged**